Terminal.Gui/Views/TextView.cs

```diff
@@ -817,25 +849,43 @@ void Adjust ()
 			}
 		}
 
-		int OffSetBackground ()
+		(int width, int height) OffSetBackground ()
 		{
-			int offB = 0;
+			int w = 0;
+			int h = 0;
 			if (SuperView?.Frame.Right - Frame.Right < 0) {
-				offB = SuperView.Frame.Right - Frame.Right - 1;
+				w = SuperView.Frame.Right - Frame.Right - 1;
 			}
-
-			return offB;
+			if (SuperView?.Frame.Bottom - Frame.Bottom < 0) {
+				h = SuperView.Frame.Bottom - Frame.Bottom - 1;
+			}
+			return (w, h);
 		}
 
 		/// <summary>
 		/// Will scroll the <see cref="TextView"/> to display the specified row at the top
 		/// </summary>
 		/// <param name="row">Row that should be displayed at the top, if the value is negative it will be reset to zero</param>
-		public void ScrollTo (int row)
+		public void ScrollToRow (int row)
```

Comment:
This is a breaking API change. Do you have any data on how many clients have used this API?

Options for addressing:
- Update the relnotes to indicate a breaking change.
- Ensure API docs indicate the API changed
- Keep the old API in place, but document it as deprecated and point to new API

Reply:
Ok I fixed it without breaking anything. Thanks.